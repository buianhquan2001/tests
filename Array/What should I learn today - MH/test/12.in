20
27
6 1 14 22 10 9 2 9 1 10 1 8 10 17 25 5 23 13 24 23 14 24 2 16 5 23 8 
24
2 4 22 19 10 11 20 16 2 16 4 17 23 24 21 17 14 4 23 11 11 3 11 1 
11
18 7 25 11 24 1 19 14 7 3 1 
4
10 13 6 9 
11
2 19 12 3 21 22 2 4 16 13 2 
31
19 15 24 16 1 12 7 3 16 14 5 25 1 13 25 20 11 9 16 10 13 11 20 13 7 9 18 10 6 14 19 
25
15 1 25 4 6 14 15 10 16 23 1 25 10 8 17 23 2 21 6 18 15 19 2 10 9 
3
25 20 2 
27
16 25 15 21 13 13 22 4 13 5 13 10 25 6 8 10 13 8 1 9 24 2 22 24 5 3 4 
30
8 9 25 24 11 12 4 12 25 6 11 16 3 15 12 18 3 21 6 3 11 25 2 25 2 21 23 6 20 17 
2
5 1 
25
10 24 11 18 17 19 9 14 9 6 22 21 17 21 19 2 7 24 8 13 25 4 3 20 4 
23
12 17 25 19 1 21 6 11 11 25 17 17 10 19 23 2 9 17 21 23 17 7 12 
25
19 9 24 12 21 10 14 25 4 9 2 9 9 21 9 12 16 21 18 19 11 24 8 21 23 
30
23 13 13 13 13 9 24 20 1 5 11 20 21 12 17 10 20 8 17 4 20 3 22 19 22 15 8 4 2 15 
7
10 19 2 11 11 12 8 
27
8 19 2 20 22 7 22 6 24 24 6 18 14 20 7 7 1 13 15 16 8 8 9 12 12 6 2 
12
25 20 9 8 5 4 1 7 8 13 8 4 
19
21 23 6 11 22 9 6 14 11 16 16 9 6 17 22 14 14 14 20 
22
13 22 4 2 8 23 10 22 15 15 21 6 13 13 3 1 14 21 3 13 2 25 
