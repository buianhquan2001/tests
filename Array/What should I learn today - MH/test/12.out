english
math
math
math
physics
physics
physics
math
physics
physics
math
physics
physics
math
math
english
math
math
english
english
